leapp/snactor/__init__.py

```diff
@@ -68,6 +79,17 @@ def cli(args):
 
 
 def main():
+    profile_enabled = os.environ.get('LEAPP_CPROFILE', '0') == '1'
```

Comment:
While I +1 the idea, I was just thinking about this and I am wondering - Is it relevant that you know it's on for others that this? This is the only use case right now, right?

Reply:
This place is the only use case so far. I was thinking if it's worth it to move it to stdlib and I'm kind of indecisive. Maybe it could be moved there if another use case appears. So feel free to dismiss this sugeestion.